mlflow/java/client/src/main/java/org/mlflow/tracking/MlflowClient.java

```diff
@@ -748,11 +748,12 @@ public String getModelVersionDownloadUri(String modelName, String version) {
    * @return A local file or directory ({@link java.io.File}) containing model artifacts
    */
   public File downloadModelVersion(String modelName, String version) {
-    String downloadUri = getModelVersionDownloadUri(modelName, version);
-
+    String scheme = "models";
```

Comment:
This should be a private member variable rather than instantiated within the function.